src/modules/sub-items/sub.items.module.js

```diff
@@ -196,22 +252,26 @@ export default class SubItemsModule extends Component {
      * @param {Object} responses
      * @memberof SubItemsModule
      */
-    updateItemsState({ locations, content, totalCount, contentTypes }) {
-        const items = locations.reduce((total, location) => {
+    updateItemsState({ page, locations, contents, totalCount, contentTypes }) {
+        const pageItems = locations.map((location) => {
             const itemLocation = location.value.Location;
+            const respectiveContent = contents.find((item) => item.value.Content._id === itemLocation.ContentInfo.Content._id).value
+                .Content;
```

Comment:
When this `.Content` is in another line it looks weird. Can we do something with this?

Reply:
I solved it by adding `map` in `loadContentItems`: 
`contents: contentInfo.View.Result.searchHits.searchHit.map((searchHit) => searchHit.value.Content)`.
Now we don't have `.Content` in another line and `contents` contains `Content`s and not `searchHit.value.Content`s.